doc/man7/OSSL_PROVIDER-FIPS.pod

```diff
@@ -426,6 +426,17 @@ A simple self test callback is shown below for illustrative purposes.
 
 =head1 NOTES
 
+Some released versions of OpenSSL do not include a validated
+FIPS provider.  To determine which versions have undergone
+the validation process, please refer to the
+L<OpenSSL Downloads page|https://www.openssl.org/source/>.  If you
```

Comment:
And is it true that 3.0.0 has been FIPS certified? Shouldn't it be 3.1.0?

Reply:
The 3.0.8 certification replaced the 3.0.0 one because it's the same certificate.
We're working on getting 3.0.0 put back on the certificate.

It's all part of the process.